### DISCUSSION TITLE
:focusable Cause Foot Terminal to be Non-interactable on Hyprland

### DISCUSSION THREAD

**Discussion by Kynson** [elkowar/eww#1055] [Q&A] [answered]
Hello,

I am experiencing a similar issue as described here: [https://www.reddit.com/r/hyprland/comments/17v012y/focusable_true_on_eww_window_makes_every_other/](https://www.reddit.com/r/hyprland/comments/17v012y/focusable_true_on_eww_window_makes_every_other/)

I updated eww to 0.5.0 on Nixos and found that if :focusable is set to true for a eww window, foot terminal (if already launched) lost its focus as soon as the eww window is open, and all newly launched windows by hyprland are non-focusable and non-interactable

Strangely, Firefox is working fine and my configuration is working before updating from 0.4.0 

```
(defwidget topBar []
    (box
        :orientaion "horizontal"
        :spacing 20
        "Test top bar"
        (button :onclick "poweroff" "Shut Down")
     )
)
(defwindow topBar
    :monitor 0
    :geometry (geometry
        :y "4%"
        :width "1248px"
        :height "4%"
        :anchor "top center"
    )
    :exclusive true
    :focusable true
    (topBar)
)
```

May I know what should I done in order to fix this? Thanks

**Comment by Dr-42** [comment, marked as answer]
As of 26/06/24 there is no fix. We have created an [issue](https://github.com/elkowar/eww/issues/651) here.
